.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,39 @@
+name: Build and test
+
+on: pull_request
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    container: ros:foxy
+
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v2
+
+      - name: Clone dependency packages
+        run: |
+          mkdir dependency_ws
+          vcs import dependency_ws < build_depends.repos
+
+      - name: Install missing dependencies
+        run: |
+          sudo apt-get update
+          rosdep update
+          DEBIAN_FRONTEND=noninteractive rosdep install --from-paths obstacle_stop_planner_refine --ignore-src --rosdistro foxy -y --skip-keys=vehicle_info_util --skip-keys=autoware_planning_msgs --skip-keys=autoware_perception_msgs --skip-keys=autoware_debug_msgs
```

Comment:
How about splitting this line into multiple lines by adding `\`?

Reply:
Like this.

```
DEBIAN_FRONTEND=noninteractive rosdep install --ignore-src --rosdistro foxy -y \
  --from-paths obstacle_stop_planner_refine \
  --skip-keys=vehicle_info_util \
  --skip-keys=autoware_planning_msgs \
  --skip-keys=autoware_perception_msgs \
  --skip-keys=autoware_debug_msgs
```

Reply:
@kenji-miyake  Thanks for the suggestion. It fixed in f4ee699

Reply:
@KeisukeShima Thank you, but let me point out one more small thing.
Actually, you need spaces before `--` to make it easier to understand where is the end of commands.

Reply:
@kenji-miyake  Thanks. fixed in eacb4a4.